lib/spectrum/request/holdings.rb

```diff
@@ -3,11 +3,16 @@
 module Spectrum
   module Request
     class Holdings
-      attr_reader :id, :focus
+      attr_reader :id, :focus, :id_field
 
       def initialize(request)
```

Comment:
I might simplify this to something like (untested!):

```ruby
 
def initialize(request)
  @request = request
  source = @request.params['source']
  id = case source
       when 'summon', 'mirlyn'
         unfiltered_id(@request)
       else
         RSolr.solr_escape(unfiltered_id(request))
       end
  id_field = case source
             when 'mirlyn'
               if id.length == 9
                 'aleph_id'
               else
                 @request.params['id_field']
               end
             end
  @query = "#{id_field}:#{id}"
end 

```

Reply:
Do we not want the solr_escaped id for mirlyn? Previously mirlyn would have been getting the solr_escaped id.